scroll-animations-1/Overview.bs

```diff
@@ -789,28 +770,66 @@ follows:
 
 </div>
 
-### The effective scroll range of a {{ScrollTimeline}} ### {#effective-scroll-range-algorithm}
-
-The procedure to calculate <dfn>effective scroll range</dfn> of a
-{{ScrollTimeline}} is as follows:
-
-1.  Run the procedure to [=resolve a scroll timeline offset=] for both {{start}}
-    and {{end}}.
-
-1.  Calculate [=effective scroll range=] as follow:
-    <div class="switch">
-    :   If [=effective start offset=] or [=effective end offset=] is null.
-    ::  The [=effective scroll range=] is null.
-
-    :   Otherwise
-    ::  The [=effective scroll range=] is the result of evaluating the following
-        expression:
-        <blockquote>
-        `[=effective end offset=] - [=effective start offset=]`
-        </blockquote>
-
-    </div>
-
+### The effective scroll offsets of a {{ScrollTimeline}} ### {#effective-scroll-offsets-algorithm}
+
+The <dfn>effective scroll offsets</dfn> are calculated by the procedure to
+<dfn>resolve scroll timeline offsets</dfn> from {{ScrollTimeline/scrollOffsets}}
+as follows:
+
+1.  Let <var>effective scroll offsets</var> be an empty list of
+    <a>effective scroll offset</a>s.
+1.  Let <var>first offset</var> be true.
+1.  If {{ScrollTimeline/scrollOffsets}} is empty
+    1.  Run the procedure to [=resolve a scroll timeline offset=] for 'auto'
+        with the <var>is first</var> flag set to <var>first offset</var> and
+        add the resulted value into <var>effective scroll offsets</var>.
+    1.  Set <var>first offset</var> to false.
+    1.  Run the procedure to [=resolve a scroll timeline offset=] for 'auto'
+        with the <var>is first</var> flag set to <var>first offset</var> and
+        add the resulted value into <var>effective scroll offsets</var>.
+1.  If {{ScrollTimeline/scrollOffsets}} has one element
+    1.  Run the procedure to [=resolve a scroll timeline offset=] for 'auto'
+        with the <var>is first</var> flag set to <var>first offset</var> and
+        add the resulted value into <var>effective scroll offsets</var>.
+    1.  Set <var>first offset</var> to false.
+1.  For each <var>scroll offset</var> in the list of
+    {{ScrollTimeline/scrollOffsets}}, perform the following steps:
+    1.  Let <var>effective offset</var> be a result of applying the procedure
```

Comment:
"Let X be _a_ result of applying Y" is pretty uncommon. (Occurs several times here). 

"Let X be _the_ result of applying Y"?

Reply:
Thanks! Done.